bears/python/VultureBear.py

```diff
@@ -1,45 +1,52 @@
-import re
-from shutil import which
-
 from coalib.bears.GlobalBear import GlobalBear
-from dependency_management.requirements.PipRequirement import PipRequirement
-from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
+from dependency_management.requirements.PipRequirement import PipRequirement
+from vulture import Vulture
+
+
+def _retrieve_unused(filenames):
```

Comment:
`_retrieve_unused` is missing an object and "retrieve" suggests that the analysis has already been done. I'd call the function `_find_unused_code` instead.